src/content/docs/ja/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -0,0 +1,608 @@
+    const pagePathname = Astro.url.pathname
+    ---
+    <Layout title="About" pathname={pagePathname}>
+        <p>About</p>
+    </Layout>
+    ```
+
+    Aboutページでは「Home」へのリンクが表示され、ホームページでは表示されません。
+</Steps>
+
+## コミュニティリソース
+
+<CardGrid>
+<LinkCard title="Migrating from Gatsby to Astro" href="https://loige.co/migrating-from-gatsby-to-astro/" 
+description="How and why I migrated this blog from Gatsby to Astro and what I learned in the process." />
+
+<LinkCard title="Migrating to Astro was EZ" href="https://joelhooks.com/migrating-to-astro-was-ez" 
+description="This is about the process of migrating from Gatsby to Astro, and why I chose Astro." />
+
+<LinkCard title="My Switch from Gatsby to Astro" href="https://www.joshfinnie.com/blog/my-switch-from-gatsby-to-astro/" 
+description="The switch to Astro is definitely worth a blog post! It’s revolutionizing the static web development scene for the better."/>
+
+<LinkCard title="Why I moved to Astro from Gatsby" href="https://dev.to/askrodney/why-i-moved-to-astro-from-gatsby-3fck" 
+description="Taking a quick look at what made me want to switch and why Astro was a good fit." />
+
+<LinkCard title="Another Migration: From Gatsby to Astro" href="https://logarithmicspirals.com/blog/migrating-from-gatsby-to-astro/" 
+description="Learn about how I transitioned my personal website from Gatsby to Astro as I share insights and experiences from the migration process."/>
+
+<LinkCard title="From Gatsby gridlock to Astro bliss: my personal site redesign" href="https://jwn.gr/posts/migrating-from-gatsby-to-astro/" 
+description="Gatsby has shown its age and I found myself seeking a modern alternative. Enter Astro — a framework that has breathed some new life into this site."/>
+
+<LinkCard title="Why and how I moved my blog away from Gatsby and React to Astro Js and Preact" href="https://www.helmerdavila.com/blog/en/why-and-how-i-moved-my-blog-away-from-gatsby-and-react-to-astro-js-and-preact" 
+description="All is about simplicity and power at the same time." />
+
+<LinkCard title="How I rewrote my HUGE Gatsby site in Astro and learned to love it in the process" href="https://dunedinsound.com/blog/how_i_rewrote_my_huge_gatsby_site_in_astro_and_learned_to_love_it_in_the_process/" 
+description="Everything is faster. Happier. More productive."/>
+
+<LinkCard title="How I switched from Gatsby to Astro (While Keeping Drupal in the Mix)" href="https://albert.skibinski.nl/en/blog/how-i-switched-gatsby-astro-while-keeping-drupal-mix/" 
+description="I came across the relatively new Astro, which ticked all the boxes."/>
+
+<LinkCard title="Migrating my website from Gatsby to Astro" href="https://dev.to/flashblaze/migrating-my-website-from-gatsby-to-astro-2ej5" 
+description="Astro has entered the chat." />
+
+<LinkCard title="Gatsby to Astro" href="https://alvin.codes/writing/gatsby-to-astro" 
+description="Why and how I migrated this website from Gatsby to Astro."/>
+
+</CardGrid>
```

Comment:
this resources are very helpful, so I’ve left them as they are.

https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/migrate-to-astro/from-gatsby.mdx?plain=1#L570-L604